components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/resources/api.yaml

```diff
@@ -476,6 +476,39 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  /retrieve-api-artifacts:
+    post:
+      summary: Get all the API runtime artifacts for a given API UUID list and gateway label
+      description: |
+        This will provide access to API runtime artifacts in database.
+      parameters:
+        - $ref: '#/parameters/requestedTenant'
+        - name: gatewayLabel
+          in: query
+          description: |
+            **Search condition**.
+             label associated with the APIs
+          type: string
+        - name: type
+          in: query
+          description: |
+            **Search condition**.
```

Comment:
Is this part required
```suggestion
```

Reply:
Fixed with 87f85e02881b3d64d82f300d60ee009702fb35d5